dvc/repo/experiments/ls.py

```diff
@@ -39,6 +43,11 @@ def ls(
         name = baseline[:7]
         if tags[baseline] or ref_heads[baseline]:
             name = tags[baseline] or ref_heads[baseline][len(base) + 1 :]
-        results[name] = [info.name for info in ref_info_dict[baseline]]
+        for info in ref_info_dict[baseline]:
+            if git_remote:
+                exp_rev = None
+            else:
+                exp_rev = next(repo.scm.branch_revs(str(info), info.baseline_sha))[:7]
```

Comment:
```suggestion
                exp_rev = scm.get_ref(str(info))
```

I think it would be better for the internal API call to return the entire rev, and for the `[:7]` shorthand truncation to be done at the UI level (in the `commands.experiments.ls` `ui.write()` calls)